peripherals/usart/_v2_with_7bit_data.yaml

```diff
@@ -0,0 +1,10 @@
+# USART v2 with 7bit data
+
+# FIXME workaround separated M0 and M1 fields
```

Comment:
Is merging M0 and M1 together as M to allow choosing between mode more simple?

- The peripheral without 9 bits implementation have only one M field.
- This will remove the M1-M0 value which is misleading.

Reply:
We can't merge non-sequential bits so we should keep them separate even though it's confusing.